src/main/java/use_case/display_episode/DisplayEpisodeInteractor.java

```diff
@@ -0,0 +1,30 @@
+package use_case.display_episode;
+
+import data_access.EpisodeDataAccess;
+import entities.Episode;
+
+import java.util.UUID;
+
+public class DisplayEpisodeInteractor implements DisplayEpisodeInputBoundary{
+
+    final EpisodeDataAccess episodeDataAccessObject;
```

Comment:
I think we should specify that these are private attributes?

Reply:
good catch! thank you